chef/cookbooks/network/recipes/default.rb

```diff
@@ -638,6 +638,14 @@ def get_datapath_id_for_ovsbridge(bridge)
     code "wicked ifup all"
     only_if { run_wicked_ifup }
   end
+  # Sometimes wicked deletes the default route upon bringing up a new interface,
+  # make sure it gets put back
+  unless ::Kernel.system("ip route |grep -q default")
+    nic = default_route[:nic]
+    gw = default_route[:gateway]
+    Chef::Log.info("Adding default route via #{gw} to #{nic}")
+    ::Kernel.system("ip route add default via #{gw} dev #{nic}")
+  end
```

Comment:
Ok, sorry, I just re-read your PR description and there you say it: no root cause known right now :frowning_face: